apps/arm/src/main.cpp

```diff
@@ -58,6 +58,9 @@ DifferentialWristController wristController(wristLeftActuator, wristRightActuato
 ClawController clawController(ArmConfig::clawActuatorConfig, clawMotor, clawEncoder, clawLimOpen, clawForceSensor,
                               clawTooltipServo, 180.0, 0.0);
 
+// Prevents PID params from being tuned over CAN inadvertently
+static bool allowPIDParamTuning = false;
```

Comment:
Not a fan of a global here. Is this shared between multiple threads?

i think itd be better design for the Controller to have a member that determines if its allowed to tune at this moment or not